.gitignore

```diff
@@ -19,6 +19,11 @@ DerivedData
 *.hmap
 *.ipa
 .env
+*.xcodeproj/*
+!*.xcodeproj/project.pbxproj
+!*.xcodeproj/xcshareddata/
+!*.xcworkspace/contents.xcworkspacedata
+/*.gcno
```

Comment:
Don't you need an ignore entry for the keys file?

Reply:
No, it is empty and I think it should be there.

Reply:
What if I decide to use it, fill it with valid credentials and commit without realising it contains sensitive data?

Reply:
Yes, you are right. 👍